software/languagetool.yml

```
@@ -0,0 +1,13 @@
+name: "LanguageTool"
+website_url: "https://languagetool.org/"
+source_code_url: "https://github.com/languagetool-org/languagetool"
+description: "LanguageTool is an Open Source proofreading software for English, Spanish, French, German, Portuguese, Polish, Dutch, and more than 20 other languages. It finds many errors that a simple spell checker cannot detect."
+licenses:
+  - LGPL-2.1
+platforms:
+  - Java
+  - Docker
+tags:
+  - Miscellaneous
+depends_3rdparty: no
+related_software_url: "https://languagetool.org/insights/post/product-windows-app/"
```

Optional so this can be removed.
```suggestion
  - Miscellaneous
related_software_url: "https://languagetool.org/insights/post/product-windows-app/"
```